### PR TITLE
Add amp-audio.md Chinese translation.

### PR DESCRIPTION
**Pull request by sss55b**
Hi,I have already translated amp-audio.md into Chinese!

### CODE REVIEW

**Comment by sss55b** [comment]
> @googlebot I signed it!

File: pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+$localization:
+  path: '/{locale}/documentation/components/amp-audio.html'
+layouts:
+  - fixed
+  - fixed-height
+  - nodisplay
+scripts:
+  - >-
+    <script async custom-element="amp-audio"
+    src="https://cdn.ampproject.org/v0/amp-audio-0.1.js"></script>
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md:26]: Please remove all keys that are not relevant for translation. I.e. only keep `teaser`, all other keys will then cascade to the original document. 🙂

File: pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+---
+
+
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (matthiasrohmer) [inline pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md:31]: Update to 2020.

**Review by matthiasrohmer — CHANGES_REQUESTED** [review]
Thanks a lot for contributing to AMP! Left two comments.

/cc @CrystalOnScript FYI

File: pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md (hunk 2 above)
Comment (sss55b) [reply pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md]: done

File: pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md (hunk 1 above)
Comment (sss55b) [reply pages/content/amp-dev/documentation/components/reference/amp-audio-v0.1@zh_CN.md]: done

**Review by matthiasrohmer — APPROVED** [review]
Thank you! /cc @patrickkettner

**Pull request merged**